src/frontend/components/SøknadsSteg/EøsSteg/IdNummer.tsx

```diff
@@ -91,47 +88,45 @@ export const IdNummer: React.FC<{
         );
     }, [idNummerFelt.verdi, idNummerFelt.valideringsstatus]);
 
+    const søkersLand = getName(landAlphaCode, valgtLocale);
     return (
         <IdNummerContainer lesevisning={lesevisning}>
             {lesevisning ? (
                 <OppsummeringFelt
-                    tittel={
-                        <SpråkTekst
-                            id={spørsmålSpråkId}
-                            values={{
-                                land: getName(landAlphaCode, valgtLocale),
-                                ...(barn && { barn: barn.navn }),
-                            }}
-                        />
-                    }
                     søknadsvar={
                         idNummerVerdiFraSøknad === AlternativtSvarForInput.UKJENT
-                            ? formatMessage(
-                                  {
-                                      id: spørsmålCheckboxSpråkId,
-                                  },
-                                  { land: getName(landAlphaCode, valgtLocale) }
-                              )
+                            ? plainTekst(spørsmålDokument.checkboxLabel, {
+                                  land: getName(landAlphaCode, valgtLocale),
```

Comment:
Ser dette er dratt ut i en variabel lenger opp `søkersLand`. Tenker også at det gjelder land for forskjellige persontyper, så kanskje variabelen kan renames til å ikke kun gjelde søker

Reply:
Enig 🚀 fiksa i  https://github.com/navikt/familie-ks-soknad/pull/78/commits/4439109072cad80be302d5f2f64641badd67576b